compiler/testData/codegen/box/primitiveTypes/kt2269.kt

```diff
@@ -1,13 +1,16 @@
 // IGNORE_BACKEND: JVM_IR
 fun box() : String {
-    230?.toByte()?.hashCode()
-    9.hashCode()
+    // Just hard enough that the test won't get optimized away at compile time.
```

Comment:
BlackBoxCodegenTestGenerated$PrimitiveTypes.testKt2269 fails: 
Unresolved reference: toInt (4,27) in /kt2269.kt

Reply:
seems // WITH_RUNTIME directive is required

Reply:
Updated with another approach so that  runtime isn't required. Also verified with :compiler:tests